setup.cfg

```diff
@@ -20,5 +20,6 @@ data_files =
     /usr/share/ansible/roles/ipaclient = roles/ipaclient/*
     /usr/share/ansible/roles/ipaconf = roles/ipaconf/*
     /usr/share/ansible/roles/ipaserver = roles/ipaserver/*
-    /usr/share/ansible/roles/krb5 = roles/krb5/*
-    /usr/share/ansible/roles/sssd = roles/sssd/*
+    /usr/share/ansible/roles/ipareplica = roles/ipareplica/*
+    /usr/share/ansible/roles/ipa-krb5 = roles/ipa-krb5/*
```

Comment:
ipa-krb5 is also not used anymore, only ipaserver, ipareplica and ipaclient should be in the list.

Reply:
You can adapt the first commit and force-push - that is accepted for a pull request.